src/main/java/io/axoniq/axonserver/connector/impl/AbstractIncomingInstructionStream.java

```diff
@@ -59,13 +61,17 @@
      * @param permits           the number of permits this stream should receive
      * @param permitsBatch      the number of permits to be consumed prior to requesting new permits
      * @param disconnectHandler a {@link Consumer} of {@link Throwable} invoked when this stream errors out
+     * @param onStartHandler    the handler to call when the upstream connection is available. Note that the call has
```

Comment:
I would've assumed `connection` instead of `call` in the second sentence here, although that might be a wrong assumption from my end.